d2themes/d2themes.go

```diff
@@ -58,3 +58,13 @@ var WarmNeutral = Neutral{
 	N6: "#ECEBEB",
 	N7: "#FFFFFF",
 }
+
+var CattpuccinMochaNeutral = Neutral{
+	N1: "#cdd6f4",
```

Comment:
capitalize plz, to be consistent

Reply:
sorry didn't see that, changed it in the theme too